resources/views/layouts/header.blade.php

```diff
@@ -20,7 +20,7 @@
 	<link rel="canonical" href="https://admin.netsoc.co" />
 	<meta property="og:title" content="UCC Netsoc Admin" />
 	<meta property="og:url" content="https://admin.netsoc.co" />
-	<meta property="og:image" content="http://files.netsoc.co/f/e9e0bee5b4/?dl=1" />
+	<meta property="og:image" content="https://files.netsoc.co/f/e9e0bee5b4/?dl=1" />
```

Comment:
There is no inherit advantage to using HTTPS over HTTP other than SSL encryption (which adds slight overhead). SSL may be wanted depending on context but that should be decided by whoever is implementing the project which is why I recommend using ```//files...``` rather than explicitly using one or the other.

That way if you wanted to run your instance of the project in HTTPS only, then having your server configured to serve HTTPS by default would make downloads of external sources HTTPS too (if the external source can serve HTTPS, otherwise depending on the browser the content won't load for security reasons)

The reason to use only one protocol throughout is so that we can take advantage of HTTP1.1's keep-alive behaviour. Using two protocols would require additional TLS handshakes for each new connection adding even more overhead.